rotkehlchen/tasks/manager.py

```diff
@@ -586,6 +589,19 @@ def _maybe_update_yearn_vaults(self) -> Optional[list[gevent.Greenlet]]:
 
         return None
 
+    def _maybe_detect_evm_accounts(self) -> Optional[list[gevent.Greenlet]]:
+        """Function that schedules EVM accounts detection task if either there is no EVM accounts
+        cache yet or this cache has expired (i.e. it's been more than a week since last update)."""
+        if should_detect_evm_accounts(self.database) is False:
+            return None
```

Comment:
Sorry I didn't notice before as I just noticed while working in my branch. This hasn't been included in the potential tasks list

Reply:
Ah wow, thanks